docs/release_notes.md

```diff
@@ -6,20 +6,24 @@
 
 - Attachment to geometry constraints in AdnMuscle and AdnRibbonMuscle solvers. Multiple target geometries are supported.
 - Slide on geometry constraints in AdnMuscle and AdnRibbonMuscle solvers. Multiple target geometries are supported.
+- Shape preservation constraints integrated in AdnMuscle, AdnRibbonMuscle, AdnSkin and AdnSimshape solvers.
 - Point mass support integrated in all solvers to control the dynamics of the system.
```

Comment:
Can you please also be verbose with the deformers here? Thanks!

Reply:
Done.